### DISCUSSION TITLE
Synology Docker and 'busy' beam.smp process

### DISCUSSION THREAD

**Discussion by paulicarmody** [plausible/analytics#748] [Self-Hosted Support]
Hi

I have Plausible running via Docker compose on my Synology NAS. All working great and checking in analytics from my website fine, however the Plausible specific docker container is constantly 'busy'. I get non stop disk activity and it seems to utilise 3-5% CPU constantly too. As soon as I switch off this container, disk and CPU activity drops.

I wouldn't expect Plausible to behave this way and only to occasionally request disk/CPU resources if/when someone visits my site.

Seams to be a beam.smp process? Would anyone know what this process is doing and how I can tame it?

<img width="1049" alt="Screenshot 2021-02-23 at 14 27 05" src="https://user-images.githubusercontent.com/2670135/108857703-44f70f00-75e3-11eb-91e2-4c41ac684038.png">

**Comment by ukutaht** [comment]
Hmm we do have a couple timers (think `setInterval`) going in the app which could use some resources while idle. We also use a background job library that might run some timers even while the app is idling. These would create some load even without traffic.

**Reply by paulicarmody** [reply]
Ok, I can understand scheduled/timed processes running at intervals but this process is running solidly, 24-7 constant disk and CPU usage.

Sadly, I'm going to have to turn off Plausible in Docker due because of this, unless there's a known  fix or workaround that someone can advise me of.
